src/containers/Interfaces/Information.js

```diff
@@ -18,7 +18,7 @@ const TAGS = [
   'thematicBreak',
 ];
 
-const getItemComponent = (item) => {
+export const getItemComponent = (item) => {
```

Comment:
Since we only care about `text` on the new interface (it's hard-coded, not dynamic), it might make sense to expose ReactMarkdown as a component (with a `content` prop) that we can use in AlterForm as 'normal' JSX. We wouldn't need this export.

Reply:
Done.